# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# T2 SDE: package/.../coreutils/disable_man.patch.cross
# Copyright (C) 2014 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; version 2 of the License. A copy of the
# GNU General Public License can be found in the file COPYING.
# --- T2-COPYRIGHT-NOTE-END ---

--- coreutils-8.22/man/local.mk.vanilla	2014-06-13 13:40:30.461044759 +0200
+++ coreutils-8.22/man/local.mk	2014-06-13 13:41:52.411041926 +0200
@@ -19,11 +19,11 @@
 EXTRA_DIST += man/help2man man/dummy-man
 
 ## Graceful degradation for systems lacking perl.
-if HAVE_PERL
-run_help2man = $(PERL) -- $(srcdir)/man/help2man
-else
+#if HAVE_PERL
+#run_help2man = $(PERL) -- $(srcdir)/man/help2man
+#else
 run_help2man = $(SHELL) $(srcdir)/man/dummy-man
-endif
+#endif
 
 man1_MANS = @man1_MANS@
 EXTRA_DIST += $(man1_MANS:.1=.x)
@@ -189,8 +189,8 @@
 	  && $(run_help2man)						\
 		     --source='$(PACKAGE_STRING)'			\
 		     --include=$(srcdir)/man/$$name.x			\
-		     --output=$$t/$$name.1 $$t/$$name			\
 		     --info-page='coreutils \(aq'$$name' invocation\(aq' \
+		     --output=$$t/$$name.1 $$t/$$name                   \
 	  && sed \
 	       -e 's|$*\.td/||g' \
 	       -e '/For complete documentation/d' \
